crypto/x509/algorithm.c

```diff
@@ -64,6 +64,15 @@
 
 #include "internal.h"
 
+/* Restrict the digests that are allowed in X509 certificates */
```

Comment:
can we do this in aws-lc? is it possible that there are customers that will be broken with this change?

Reply:
discussed internally, resolving the conversation